homeassistant/components/garages_amsterdam/entity.py

```diff
@@ -22,3 +23,8 @@ def __init__(
         self._attr_unique_id = f"{garage_name}-{info_type}"
         self._garage_name = garage_name
         self._info_type = info_type
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, garage_name)},
```

Comment:
Third would create a separate service "device" for each garage. Does that make sense? Its one online service afterall?

Reply:
Yes, technically it's one service where we get data from, but this way we can add entity translations to every garage. I need to double check but afaik there's one garage per config entry so this makes the most sense imo

Reply:
Almost ends up a standard device then. It does reflect the garage state. Not the service. A bit of a gray zone. I think Im okey with it at least.